### PR TITLE
e2e test: Fix flaky test increasing timeout when changing replicas number

### PR DESCRIPTION
**Pull request by fossedihelm**
**What this PR does / why we need it**:
We saw that the pods were terminating and we could see an event of successful delete right after the timeout.
Test results flaky due to the short timeout.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

/cc @xpivarc

### CODE REVIEW

**Comment by EdDev** [comment]
>  Fixes # https://issues.redhat.com/browse/CNV-23238

The ref is not publicly accessible.

The change itself seems reasonable to me, but could you please share why you believe that increasing the timeout will solve this? E.g. what tests have been done to show this helps?

File: tests/operator_test.go [hunk 1]
```diff
@@ -2735,7 +2735,7 @@ spec:
 						}
 					}
 					return true
-				}, 60*time.Second, 1*time.Second).Should(BeTrue())
+				}, 120*time.Second, 1*time.Second).Should(BeTrue())
```
[the remark this answers is not in the document]
Comment (xpivarc) [reply tests/operator_test.go:2738]: Indeed that would be much helpful!

**Comment by xpivarc** [comment]
> > Fixes # https://issues.redhat.com/browse/CNV-23238
> 
> The ref is not publicly accessible.
> 
> The change itself seems reasonable to me, but could you please share why you believe that increasing the timeout will solve this? E.g. what tests have been done to show this helps?

I can fill the gaps. We saw that the pods were terminating and we could see an event of successful delete right after the timeout.

File: tests/operator_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (fossedihelm) [reply tests/operator_test.go]: Thanks for raising this. I pushed the changes! PTAL

**Comment by EdDev** [comment]
> > > Fixes # https://issues.redhat.com/browse/CNV-23238
> > 
> > 
> > The ref is not publicly accessible.
> > The change itself seems reasonable to me, but could you please share why you believe that increasing the timeout will solve this? E.g. what tests have been done to show this helps?
> 
> I can fill the gaps. We saw that the pods were terminating and we could see an event of successful delete right after the timeout.

Thank you.
Can we add this info and remove the non-public ref then?

**Comment by fossedihelm** [comment]
> > > > Fixes # https://issues.redhat.com/browse/CNV-23238
> > > 
> > > 
> > > The ref is not publicly accessible.
> > > The change itself seems reasonable to me, but could you please share why you believe that increasing the timeout will solve this? E.g. what tests have been done to show this helps?
> > 
> > 
> > I can fill the gaps. We saw that the pods were terminating and we could see an event of successful delete right after the timeout.
> 
> Thank you. Can we add this info and remove the non-public ref then?

Sure! Done. Thanks @EdDev

**Review by xpivarc — COMMENTED** [review]
/approve

**Pull request merged**